kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/utils/NamespaceUtils.java

```diff
@@ -55,6 +55,10 @@ public static void deleteNamespaceWithoutWait(String namespace) {
     private static void deleteNamespace(String namespace, boolean wait) {
         if (!Environment.SKIP_TEARDOWN) {
             LOGGER.info("Deleting namespace: {}", namespace);
+            if (namespace.equals(Environment.STRIMZI_NAMESPACE) && Environment.SKIP_STRIMZI_INSTALL) {
+                LOGGER.info("Skipped namespace: {}, because SKIP_STRIMZI_INSTALL is true", namespace);
+                return;
+            }
```

Comment:
I think we have the division of responsibilities between the tests and the test framework wrong here. This kind of tear down logic should be part of the framework. 

That said thats a change for another day.